src/YUILoader.cc

```diff
@@ -120,7 +122,16 @@ void YUILoader::loadUI( bool withThreads )
 
 	try
 	{
-	    loadPlugin( wantedGUI, withThreads );
+            // Load integration testing framework plugin, which load required UI
+            if ( strcmp( envTestEnable, "1" ) == 0 )
```

Comment:
As the GTK UI is not supported yet we should at least skip the testing framework in that case. Something like
```suggestion
            if ( strcmp( envTestEnable, "1" ) == 0 && wantedGUI != YUIPlugin_Gtk)
```